es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,54 @@
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```

Comment:
The `lib` folder has an `index.js` file that uses named exports for all the functions in the `lib` folder. So here you can import multiple things from `lib` on one line:
```JAVASCRIPT
import { createElement, noop } from '../lib';
```